xds/internal/xdsclient/controller/transport.go

```diff
@@ -388,26 +388,34 @@ func (t *Controller) reportLoad(ctx context.Context, cc *grpc.ClientConn, opts c
 
 		retries++
 		lastStreamStartTime = time.Now()
-		stream, err := t.vClient.NewLoadStatsStream(ctx, cc)
+		// streamCtx is created and canceled in case we terminate the stream
+		// early for any reason, to avoid gRPC-Go leaking the RPC's monitoring
+		// goroutine.
```

Comment:
RPC's monitoring goroutine? Which goroutine are you taking about here?

Reply:
This one:

https://github.com/grpc/grpc-go/blob/64174955202ffb5ea4122e25d1aaece49cc5a3ed/stream.go#L354-L361